generic/tclsass.c

```diff
@@ -1420,7 +1422,12 @@ static int SassObjCmd(
 
 done:
     if (optsPtr != NULL) {
-	free(optsPtr); /* HACK: No official destructor function. */
```

Comment:
IHMO it is actually the *correct* code for 3.4.x and below.  There are multiple issues on the upstream project for memory leaks.  The responses are typically caller frees until sass_delete_xxx appears.  The code has the ifdef around it with the correct future API  code.  But it is lacking the autoconf detect.